docs/fast_data/configuration/config_maps/projection_changes_schema.md

```diff
@@ -47,12 +47,17 @@ The Projection Changes Schema is made of the following fields:
 </details>
 
 :::note
-All the keys in uppercase are values that you must change depending on your data, while the keys in lowercase are keywords that should not be changed
+All the keys in uppercase are values that you must change depending on your data, while the keys in lowercase are keywords that should not be changed.
 :::
 
 In some cases you may want a finer control over the creation of the projection changes identifier. Such control can be achieved within service configuration providing a _custom function_,
 which is applied to each document retrieved by the last step of the strategy path (in this case records extracted from `BASE_PROJECTION` collection).
-The custom function file can be loaded as a config map of the service, while in the `projectionChangesSchema` configuration file
+
+:::caution
+Please note that this feature is not supported by Real-Time Updater.
+:::
```

Comment:
There are lots of projects with RTU that count on this feature. This is currently supported by the RTU, just not with the async generators. Are you sure you want to make that claim?

Reply:
Semi-automatic strategies are currently not supported by RTU and they won't be, since it employs a strategy resolutor different from the one that can be found in the SVTG.